runtime/domain.c

```diff
@@ -932,32 +924,57 @@ void caml_request_minor_gc (void)
 static void caml_poll_gc_work()
 {
   CAMLalloc_point_here;
+  { // No GC in this block
```

Comment:
I didn't understand this comment, as there is a call to `caml_empty_minor_heaps_once` inside the block!

Reply:
I'm unsure as well, I think it is only meant for the first part of this function? (and a Gc should only be triggered after this preambule, through the `need_minor_gc` flag?) @sadiqj

Reply:
I've read that comment a few times and I don't understand what it means. Suggest we get rid of it.